spec/embiggen/uri_spec.rb

```diff
@@ -95,6 +95,13 @@ module Embiggen
         expect { uri.expand }.to raise_error(BadShortenedURI)
       end
 
+      it 'raises and error if the URI returned is not valid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.